recipes/newrelic/infrastructure/ohi/rabbitmq/rhel.yml

```diff
@@ -135,20 +132,20 @@ install:
             - name: rabbitmq
               command: all
               arguments:
-                      hostname: {{.NR_CLI_HOSTNAME}}
-                      port: {{.NR_CLI_PORT}}
-                      username: {{.NR_CLI_USERNAME}}
-                      password: {{.NR_CLI_PASSWORD}}
-                      ca_bundle_dir: {{.NR_CLI_BUNDLE_DIR}}
-                      ca_bundle_file: {{.NR_CLI_BUNDLE_FILE}}
-                      config_path: {{.NR_CLI_CONF_PATH}}
-                      use_ssl: {{.NR_CLI_SSL}}
-                      queues: '["{{.NR_CLI_QUEUES}}"]'
-                      queues_regexes:
-                      exchanges: '["{{.NR_CLI_EXCHANGES}}"]'
-                      exchanges_regexes: 
-                      vhosts: '["{{.NR_CLI_VHOST}}"]'
-                      vhosts_regexes: 
+                hostname: {{.NR_CLI_HOSTNAME}}
+                port: {{.NR_CLI_PORT}}
+                username: {{.NR_CLI_USERNAME}}
+                password: {{.NR_CLI_PASSWORD}}
+                ca_bundle_dir: {{.NR_CLI_API_CA_BUNDLE_DIR}}
+                ca_bundle_file: {{.NR_CLI_API_CA_BUNDLE_FILE}}
+                use_ssl: {{.NR_CLI_SSL}}
+                config_path: {{.NR_CLI_RABBIT_CONFIG_PATH}}
+                queues: '{{.NR_CLI_QUEUES}}'
+                queues_regexes:
+                exchanges: '{{.NR_CLI_EXCHANGES}}'
+                exchanges_regexes:
+                vhosts: '{{.NR_CLI_VHOSTS}}'
+                vhosts_regexes:
           EOT
           fi
```

Comment:
@jbeveland27 I have just notice that you're making some changes. The reason (propably) why checks didn't pass is that you have to change the exporting variables (`NR_CLI_CONF_PATH` ->` NR_CLI_RABBIT_CONFIG_PATH ` and `NR_CLI_VHOST` -> `NR_CLI_VHOSTS `) in the main.yml file. Can't contact with you in different way right now :(. I hope that will help :)

Reply:
Ah, that makes sense! Thanks for the quick feedback 😄